resources/com.sap.piper/pipeline/stageDefaults.yml

```diff
@@ -8,6 +8,11 @@ stages:
     stepConditions:
       sonarExecuteScan:
         filePattern: '**/sonar-project.properties'
+      mavenExecuteIntegration:
+        filePattern: 'integration-tests/pom.xml'
+      npmExecuteScripts:
+        npmScripts:
+          - 'ci-frontend-unit-test'
```

Comment:
The steps "mavenExecuteStaticCodeChecks" and "npmExecuteLint" are expected to be activated via manual config only, correct?

Reply:
Yes they have to be activated manually via stage configuration.

Reply:
if `ci-frontend-unit-test` is to be configured by the user, maybe we should activate the step rather in case npmExecuteScripts.runScripts is configured.

Reply:
Good point, makes sense. I will adapt it accordingly.